imageregistry/v1/types.go

```diff
@@ -303,8 +302,32 @@ type ImageRegistryConfigStorageAzure struct {
 	// object.
 	// +optional
 	CloudName string `json:"cloudName,omitempty"`
+	// networkAccess defines the network access level for the storage account.
+	// "Internal" means the storage account will be private, "External" means
+	// the storage account will be publicly accessible.
+	// Defaults to "External".
+	// +kubebuilder:validation:Enum="Internal";"External"
+	// +kubebuilder:default="External"
+	// +optional
+	NetworkAccess AzureNetworkAccessibility `json:"networkAccess,omitempty"`
+	// vnetName is the name of the vnet the registry operates in.
+	// +optional
+	VNetName string `json:"vnetName,omitempty"`
```

Comment:
Is there any specific regex for vnet name? Or, any string is acceptable? Is there a minimal length for the name?

Reply:
There might something on the Azure api side. We're not creating a vnet here though, when it's set here it's already been create and is in use by the cluster